manifests/proxy.pp

```diff
@@ -108,6 +108,17 @@
 # [*database_port*]
```

Comment:
the permissions on this file change
that's probably a mistake

Reply:
Good pickup - thanks. I'll send through a commit to fix this later today.